src/display-x11.cc

```diff
@@ -227,6 +230,29 @@ bool display_output_x11::shutdown() {
   return true;
 }
 
+inline void propagate_unconsumed_event(XEvent &ev, bool is_consumed) {
```

Comment:
This function was tested thoroughly on KDE plasma with background widgets and such.

Reply:
This code portion is now additionally invoked on cursor movement so that e.g. desktop shortcuts get proper on-hover highlighting.